magento/pwa-studio
# [PWA-2201] Add Cypress test coverage: VeniaCartPageEditCartAndCheckout.xml

## Description

PR to deprecate MFTF test in favor of Cypress test.

### Changes

- Converts [VeniaCartPageEditCartAndCheckout](https://github.com/magento-commerce/pwa-tests/blob/sampledata_develop/app/code/Magento/Pwa/Test/Mftf/Test/VeniaCartPageEditCardAndCheckout.xml) MFTF test into a Cypress test.
- MFTF test will be deleted in https://github.com/magento-commerce/pwa-tests/pull/36

## Related Issue







Closes PWA-2201

## Acceptance





### Verification Stakeholders



### Specification




## Verification Steps






#### Test scenario(s) for direct fix/feature




1.  Run created cypress tests in PR instance. Test file is `venia-integration-tests/src/tests/e2eTests/cartPage/editCartAndCheckout.spec.js`
2. Assure test completes without any fails
3. Check test steps in https://jira.corp.magento.com/browse/PWA-1213, verify that all steps are covered.

#### Test scenario(s) for any existing impacted features/areas





#### Test scenario(s) for any Magento Backend Supported Configurations






#### Is Browser/Device testing needed?





#### Any ad-hoc/edge case scenarios that need to be considered?




## Screenshots / Screen Captures (if appropriate)

## Breaking Changes (if any)





## Checklist
